### ISSUE TITLE
The release is not tagged and release until I run  release-please github-release...

### ISSUE THREAD

**Issue by oktayalizada** [google-github-actions/release-please-action#939]
### TL;DR

Pull request created and approved changelog is changed .release-please-manifest.json is also changed. But release and tag is not happening.  After running  the following command the release and tag is generated which supposed me generated automatically: 
`release-please github-release  --token=some_token --repo-url=some_repo`

### Expected behavior

Release happens after merge of pull request

### Observed behavior

Changelog and .release-please-manifest.json are changed. Pull request merged but tag and release is not happening.

### Action YAML

```yaml
name: ci

permissions:
  contents: write
  pull-requests: write

on:
  push:
    branches:
      - "*"
    paths-ignore:
      - "**.md"

jobs:
  test:
    runs-on: windows-latest
    strategy:
      matrix:
        go-version: [ '1.21.x' ]
    steps:
      - uses: actions/checkout@v4
      - name: Setup Go ${{ matrix.go-version }}
        uses: actions/setup-go@v5
        with:
          go-version: ${{ matrix.go-version }}

          
      - name: Install dependencies
        run: go get .

      - name: Test with the Go CLI
        run: go test ./...

  lint:
    name: lint
    runs-on: windows-latest
    steps:
      - uses: actions/checkout@v4
      - uses: actions/setup-go@v5
        with:
          go-version: '1.21'
          cache: false
                    
      - name: Install dependencies
        run: go get .
        
      - name: golangci-lint
        uses: golangci/golangci-lint-action@c67416616c29c3c48d26b59c45cadb56966d80aa
        with:
          version: v1.54
          args: --timeout 3m --verbose
          
  release:
    if: github.ref == 'refs/heads/main' && startsWith(github.event.head_commit.message, 'bump')
    needs: [test, lint]
    runs-on: ubuntu-latest
    steps:
      - name: Generate a token
        id: generate_token
        uses: actions/create-github-app-token@v1
        with:
          app-id: ${{ secrets.APP_ID }}
          private-key: ${{ secrets.APP_PRIVATE }}
          
      - uses: google-github-actions/release-please-action@v4
        with:
          token: "${{ steps.generate_token.outputs.token }}"
          manifest-file: .release-please-manifest.json
          config-file: release-please-config.json
```

### Log output

```text
##[debug]Evaluating condition for step: 'Run google-github-actions/release-please-action@v4'
##[debug]Evaluating: success()
##[debug]Evaluating success:
##[debug]=> true
##[debug]Result: true
##[debug]Starting: Run google-github-actions/release-please-action@v4
##[debug]Loading inputs
##[debug]Evaluating: steps.generate_token.outputs.token
##[debug]Evaluating Index:
##[debug]..Evaluating Index:
##[debug]....Evaluating Index:
##[debug]......Evaluating steps:
##[debug]......=> Object
##[debug]......Evaluating String:
##[debug]......=> 'generate_token'
##[debug]....=> Object
##[debug]....Evaluating String:
##[debug]....=> 'outputs'
##[debug]..=> Object
##[debug]..Evaluating String:
##[debug]..=> 'token'
##[debug]=> '***'
##[debug]Result: '***'
##[debug]Evaluating: github.repository
##[debug]Evaluating Index:
##[debug]..Evaluating github:
##[debug]..=> Object
##[debug]..Evaluating String:
##[debug]..=> 'repository'
##[debug]Evaluating: github.api_url
##[debug]Evaluating Index:
##[debug]..Evaluating github:
##[debug]..=> Object
##[debug]..Evaluating String:
##[debug]..=> 'api_url'
##[debug]=> 'https://api.github.com'
##[debug]Result: 'https://api.github.com'
##[debug]Evaluating: github.graphql_url
##[debug]Evaluating Index:
##[debug]..Evaluating github:
##[debug]..=> Object
##[debug]..Evaluating String:
##[debug]..=> 'graphql_url'
##[debug]=> 'https://api.github.com/graphql'
##[debug]Result: 'https://api.github.com/graphql'
##[debug]Loading env
Run google-github-actions/release-please-action@v4
Running release-please version: 16.5.0
##[debug]Loading manifest from config file
❯ Fetching release-please-config.json from branch main
❯ Fetching .release-please-manifest.json from branch main
##[debug]Creating pull requests
✔ Building releases
✔ Building strategies by path
❯ .: go
##[debug]Loading manifest from config file
❯ Fetching release-please-config.json from branch main
❯ Fetching .release-please-manifest.json from branch main
##[debug]Creating pull requests
✔ Building pull requests
✔ Building strategies by path
❯ .: go
✔ Collecting release commit SHAs
❯ release search depth: 400
❯ Fetching releases with cursor undefined
❯ Found release for path ., v0.0.1
❯ release for path: ., version: 0.0.1, sha: something
✔ Collecting commits since all latest releases
❯ commit search depth: 500
❯ Set(1) { 'something' }
❯ Fetching merge commits on branch main with cursor: undefined
❯ Backfilling file list for commit: something
❯ Found 2 files
❯ Backfilling file list for commit: something
❯ Found 2 files
❯ Backfilling file list for commit: something
❯ Found 1 files
❯ Backfilling file list for commit: something
❯ Found 2 files
❯ Backfilling file list for commit: something
❯ Found 2 files
❯ Backfilling file list for commit: something
❯ Found 2 files
❯ Backfilling file list for commit: something
❯ Found 2 files
❯ Backfilling file list for commit: something
❯ Found 1 files
❯ Backfilling file list for commit: something
❯ Found 1 files
❯ Backfilling file list for commit: something
❯ Found 1 files
❯ Backfilling file list for commit: something
❯ Found 2 files
❯ Backfilling file list for commit: something
❯ Found 1 files
❯ Backfilling file list for commit: something
❯ Found 2 files
❯ Backfilling file list for commit: something
❯ Found 1 files
❯ Backfilling file list for commit: something
❯ Found 0 files
❯ Backfilling file list for commit: something
❯ Found 0 files
❯ Backfilling file list for commit: something
❯ Found 2 files
❯ Backfilling file list for commit: something
❯ Found 2 files
❯ Backfilling file list for commit: something
❯ Found 1 files
❯ Backfilling file list for commit: something
❯ Found 1 files
❯ Backfilling file list for commit: something
❯ Found 2 files
❯ Backfilling file list for commit: something
❯ Found 2 files
❯ Backfilling file list for commit: something
❯ Found 4 files
❯ Backfilling file list for commit: something
❯ Found 1 files
❯ Backfilling file list for commit: something
❯ Found 1 files
✔ Splitting 10 commits by path
✔ Building candidate release pull request for path: .
❯ type: go
❯ targetBranch: main
❯ commit could not be parsed: somethingMerge pull request #47 from repo/release-please--branches--main
❯ commit could not be parsed: something Merge pull request #46 from repo/release-please--branches--main
❯ commits: 8
✔ Considering: 8 commits
❯ component: 
❯ pull request title pattern: undefined
✔ Looking for open release pull requests
✔ found 0 open release pull requests.
✔ Looking for snoozed release pull requests
✔ found 0 snoozed release pull requests.
❯ Fetching CHANGELOG.md from branch main
❯ Fetching .release-please-manifest.json from branch main
✔ Starting GitHub PR workflow...
✔ Successfully found branch HEAD sha "something".
✔ Successfully created branch at https://api.github.com/repos/something/git/refs/heads/release-please--branches--main
✔ Got the latest commit tree
✔ Successfully created a tree with the desired changes with SHA 6015f77acc4bc08f724d62516f0c6b462500c1b9
✔ Successfully created commit. See commit at https://api.github.com/repos/repo/git/commits/something
✔ Updating reference heads/release-please--branches--main to something
✔ Successfully updated reference release-please--branches--main to 7f4890bbc4154144593d251780ec0a9b446db439
✔ Successfully opened pull request available at url: https://api.github.com/repos/something/pulls/48.
✔ Successfully opened pull request: 48.
✔ Successfully added labels autorelease: pending to issue: 48
##[debug]Node Action run completed with exit code 0
##[debug]Set output releases_created = false
##[debug]Set output paths_released = []
##[debug]Set output prs_created = true
##[debug]Set output pr = {"headBranchName":"release-please--branches--main","baseBranchName":"main","number":48,"title":"chore(main): release 0.0.2","body":":robot: I have created a release *beep* *boop*\n---\n\n\n## [0.0.2](https://github.com/something/compare/v0.0.1...v0.0.2) (2024-02-06)\n\n\n### Bug Fixes\n\n* add lint and test before release ([9124fc5](https://github.com/something/commit/something))\n* change version file to release-please-manifest-file ([362920c](https://github.com/something/commit/something))\n\n---\nThis PR was generated with [Release Please](https://github.com/googleapis/release-please). See [documentation](https://github.com/googleapis/release-please#release-please).","files":[],"labels":["autorelease: pending"]}
##[debug]Set output prs = [{"headBranchName":"release-please--branches--main","baseBranchName":"main","number":48,"title":"chore(main): release 0.0.2","body":":robot: I have created a release *beep* *boop*\n---\n\n\n## [0.0.2](https://github.com/something/compare/v0.0.1...v0.0.2) (2024-02-06)\n\n\n### Bug Fixes\n\n* add lint and test before release ([9124fc5](https://github.com/something/commit/something))\n* change version file to release-please-manifest-file ([362920c](https://github.com/something/commit/something))\n\n---\nThis PR was generated with [Release Please](https://github.com/googleapis/release-please). See [documentation](https://github.com/googleapis/release-please#release-please).","files":[],"labels":["autorelease: pending"]}]
##[debug]Finishing: Run google-github-actions/release-please-action@v
```

### Additional information

release-please-config.json:

`{
  "packages": {
    ".": {
      "changelog-path": "CHANGELOG.md",
      "release-type": "go",
      "bump-minor-pre-major": false,
      "bump-patch-for-minor-pre-major": false,
      "draft": false,
      "prerelease": false
    }
  },
  "$schema": "https://raw.githubusercontent.com/googleapis/release-please/main/schemas/config.json"
}`

.release-please-manifest.json

`{
  ".": "0.0.2"
}`

**Comment by shoootyou** [comment]
@oktayalizada after that, will be a new pull request with the release phase as showing in your log:

`✔ Successfully opened pull request available at url: https://api.github.com/repos/something/pulls/48.`
`✔ Successfully opened pull request: 48.`

after that approve that pull request and it will create the release

**Comment by erikgb** [comment]
I think the problem is that release-please (or GitHub) doesn't **create** release-please labels to put on release-please PRs. After manually **creating** the release-please labels, release-please is **actually** adding these labels to the release-please PRs.

Seems related to https://github.com/googleapis/release-please/issues/2465

**Comment by preet248** [comment]
@erikgb , what to do in this case?

**Comment by erikgb** [comment]
@preet248 As a workaround you can:

1. Manually create the labels (see image below)
2. Re-run the release-please workflow

Ensure that the past step adds the `autorelease:pending` label to the release PR before merging it to cut the release. This worked for me/us.

![Image](https://github.com/user-attachments/assets/22c4d79a-9d80-40d5-81fb-e148b4ff38e3)
